src/ImageRunModal.jsx

```diff
@@ -366,12 +366,6 @@ export class ImageRunModal extends React.Component {
         if (value.length < 2)
             return;
 
-        // Don't search for a value with a tag specified
-        const patt = /:[\w|\d]+$/;
```

Comment:
This means "the tag name can contain a literal `|`" which I believe is not true. A regexp character group is just a simple concatenation, i.e. `[\w\d]`.